docs/source/data/subscriptions.mdx

```diff
@@ -540,6 +562,22 @@ The following are community-created `PubSub` libraries for popular event-publish
 
 If none of these libraries fits your use case, you can also create your own `PubSubEngine` subclass. If you create a new open-source library, click **Edit on GitHub** to let us know about it!
 
-## The `graphql-ws` transport library
+## Old `subscriptions-transport-ws` library
+
+>If you use subscriptions with Apollo Client you must ensure **[both your client and server subscription protocols are consistent](https://www.apollographql.com/docs/react/data/subscriptions/#choice-of-subscription-protocol)** for the subscription library you are using.
+
+This article details how to set up the [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) library to add support for subscriptions to Apollo Server. Previously this article recommended using the `subscriptions-transport-ws` library; however, this library is no longer actively maintained.
```

Comment:
details -> describes (I think we should only use "details" as a noun for more predictability)

Reply:
Actually, I think we can cut the first sentence entirely, and tweak the rest to:

This article previously demonstrated using the `subscriptions-transport-ws` library to set up subscriptions. However, this library is no longer actively maintained. You can still use it with Apollo Server, but we strongly recommend using `graphql-ws` instead.